test/integration/render-tests/map-projections/integrations/set-projection/set-projection-precedence/style.json

```diff
@@ -0,0 +1,43 @@
+{
+    "version": 8,
+    "metadata": {
+      "test": {
+        "width": 512,
+        "height": 512,
```

Comment:
CI is failing on a minor rendering difference (maybe between operating systems).
<img width="1276" alt="image" src="https://user-images.githubusercontent.com/14878684/167947717-44181b46-3eac-440c-8f60-90d9127e0146.png">

This can be fixed by adding tolerance:

```suggestion
        "height": 512,
        "allowed": .001
```